src/index.html

```diff
@@ -5,9 +5,46 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
+
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__img"></div>
+
+      <div class="card__info">
+        <div class="card__title">APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</div>
+        <div class="card__code">Product code: 195434</div>
+      </div>
+
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__rate">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__priceText">Price:</div>
+        <div class="card__amount">$2,199</div>
+      </div>
+
+      <button
+        class="card__button"
+        name="card__button"
+        type="submit"
+        data-qa="hover"
+        >
```

Comment:
```suggestion
      <button
        class="card__button"
        name="card__button"
        type="submit"
        data-qa="hover"
      >
```
closing bracket should be on the same level as opening one